src/services/decoders/utils.ts

```
@@ -0,0 +1,62 @@
+ * @param supportsAutoGeneratedKeys
+ * @return An object containing the parsed log level key and timestamp key.
+ * @throws {Error} If the keys contain reserved symbols.
+ */
+const parseFilterKeys = (decoderOptions: DecoderOptions, supportsAutoGeneratedKeys: boolean): {
+    logLevelKey: ParsedKey;
+    timestampKey: ParsedKey;
+} => {
+    const parsedLogLevelKey = preprocessThenParseFilterKey(decoderOptions.logLevelKey);
+    const parsedTimestampKey = preprocessThenParseFilterKey(decoderOptions.timestampKey);
+
+    if (false == supportsAutoGeneratedKeys &&
```

ESLint

[error] Expected '===' and instead saw '=='.

(eqeqeq)